packages/bpk-component-text/src/BpkText-test.js

```diff
@@ -46,7 +46,7 @@ describe('BpkText', () => {
       </BpkText>,
     );
     expect(asFragment()).toMatchSnapshot();
-    expect(consoleWarnFn.mock.calls.length).toBe(1);
+    expect(consoleWarnFn.mock.calls.length).toBe(2);
```

Comment:
oh good point - it's because of the weight property as it still has a default value so it will actually always trigger a warning it seems. I thought it was actually triggered because of the function call because the test is using bold!
So we might need to change the default to null then and make sure we check that it exists before we use it

Reply:
I actually added a test just to check and there is a warning even if there is no bold or weight property given and I think it's because the weight property has a default so we'll need to change that otherwise everyone using BpkText will get warnings 😄